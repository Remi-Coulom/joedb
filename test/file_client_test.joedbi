transaction
 create_table person
 add_field person name string
 insert_into person 0 "toto"
 insert_into person 0 "titi"
 insert_into person 1 "titi"
 table person
quit
db
 schema
 table person
quit
