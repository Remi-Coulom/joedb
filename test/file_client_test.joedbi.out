[36mwritable_client(41)> [0m[36mwritable_client(41)/transaction> [0mOK:  create_table person
[36mwritable_client(41)/transaction> [0mOK:  add_field person name string
[36mwritable_client(41)/transaction> [0mOK:  insert_into person 0 "toto"
[36mwritable_client(41)/transaction> [0mException caught: insert: record_id already in use
Line 4:  insert_into person 0 "titi"
[36mwritable_client(41)/transaction> [0mOK:  insert_into person 1 "titi"
[36mwritable_client(41)/transaction> [0m--------
  name  
--------
0 "toto"
1 "titi"
OK:  table person
[36mwritable_client(41)/transaction> [0mOK: quit
OK: transaction
[36mwritable_client(75)> [0m[36mwritable_client(75)/db> [0mcreate_table person
add_field person name string
OK:  schema
[36mwritable_client(75)/db> [0m--------
  name  
--------
0 "toto"
1 "titi"
OK:  table person
[36mwritable_client(75)/db> [0mOK: quit
OK: db
[36mwritable_client(75)> [0m